src/components/radio-group/radio/bl-radio.ts

```diff
@@ -0,0 +1,138 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { event, EventDispatcher } from '../../../utilities/event';
+import type BlRadioGroup from '../bl-radio-group';
+import { blChangeEventName, blRadioGroupTag } from '../bl-radio-group';
+import style from './bl-radio.css';
+
+
+export const blRadioTag = 'bl-radio';
+
+export const blCheckedEventName = 'bl-checked';
+
+/**
+ * @tag bl-radio
+ * @summary Baklava Radio Option component
+ */
+@customElement(blRadioTag)
+export default class BlRadio extends LitElement {
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+
+  @property()
+  name: string;
+
+  @property()
+  value: string;
+
+  /**
+   * Sets option as disabled
+   */
+  @property({ type: Boolean, reflect: true })
```

Comment:
The selected and disabled bl-radio does not match with the one in figma

Storybook:
![image](https://user-images.githubusercontent.com/70342517/201657220-6afbe31d-b8b8-4af6-b6b8-b5ac95e6fb7a.png)

Figma:
![image](https://user-images.githubusercontent.com/70342517/201657480-ae81230f-2f01-4dea-b6f4-e3ebf06100ec.png)

Is this on purpose @muratcorlu ?

Reply:
Hmm, good catch. I'll fix and add a story for this case.